client/client.go

```diff
@@ -343,16 +353,43 @@ func (c *Client) processChannelRequests(ctx context.Context) {
 				return
 			}
 
-			c.runner.Start(action, c.getActionTimeout(p))
+			c.actionsRunner.Start(action, c.getActionTimeout(p))
 			responsePayload = new(agentpb.StartActionResponse)
 
 		case *agentpb.StopActionRequest:
-			c.runner.Stop(p.ActionId)
+			c.actionsRunner.Stop(p.ActionId)
 			responsePayload = new(agentpb.StopActionResponse)
 
 		case *agentpb.CheckConnectionRequest:
 			responsePayload = c.connectionChecker.Check(ctx, p, req.ID)
 
+		case *agentpb.StartJobRequest:
+			var job jobs.Job
```

Comment:
maybe also extract these code to function?
I don't like case inside case. But it's just my personal opinion.

Reply:
Done